### PR TITLE
Update documentation for release v0.0.10

### PR DESCRIPTION
**Pull request by cw75**

### CODE REVIEW

File: api-reference/sdk-reference/package-aqueduct/README.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
 * [`error`](https://docs.aqueducthq.com/api-reference/sdk-reference/package-aqueduct/aqueduct.error)
 * [`flow`](https://docs.aqueducthq.com/api-reference/sdk-reference/package-aqueduct/aqueduct.flow)
-* [`generic_artifact`](https://docs.aqueducthq.com/api-reference/sdk-reference/package-aqueduct/aqueduct.generic_artifact)
-* [`metric_artifact`](https://docs.aqueducthq.com/api-reference/sdk-reference/package-aqueduct/aqueduct.metric_artifact)
 * [`operators`](https://docs.aqueducthq.com/api-reference/sdk-reference/package-aqueduct/aqueduct.operators)
 * [`schedule`](https://docs.aqueducthq.com/api-reference/sdk-reference/package-aqueduct/aqueduct.schedule)
-* [`table_artifact`](https://docs.aqueducthq.com/api-reference/sdk-reference/package-aqueduct/aqueduct.table_artifact)
-* [`param_artifact`](https://docs.aqueducthq.com/api-reference/sdk-reference/package-aqueduct/aqueduct.param_artifact)
-###\ package aqueduct.constants
+### package aqueduct.artifacts
```
Comment (vsreekanti) [inline api-reference/sdk-reference/package-aqueduct/README.md:9]: Can you add a new line before this?

File: api-reference/sdk-reference/package-aqueduct/README.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
+* [`bool_artifact`](https://docs.aqueducthq.com/api-reference/sdk-reference/package-aqueduct/package-aqueduct.artifacts/aqueduct.artifacts.bool_artifact)
+* [`generic_artifact`](https://docs.aqueducthq.com/api-reference/sdk-reference/package-aqueduct/package-aqueduct.artifacts/aqueduct.artifacts.generic_artifact)
+* [`numeric_artifact`](https://docs.aqueducthq.com/api-reference/sdk-reference/package-aqueduct/package-aqueduct.artifacts/aqueduct.artifacts.numeric_artifact)
+* [`param_artifact`](https://docs.aqueducthq.com/api-reference/sdk-reference/package-aqueduct/package-aqueduct.artifacts/aqueduct.artifacts.param_artifact)
+* [`table_artifact`](https://docs.aqueducthq.com/api-reference/sdk-reference/package-aqueduct/package-aqueduct.artifacts/aqueduct.artifacts.table_artifact)
+### package aqueduct.constants
 * [`constants.exports`](https://docs.aqueducthq.com/api-reference/sdk-reference/package-aqueduct/package-aqueduct.constants/aqueduct.constants.exports)
 * [`constants.metrics`](https://docs.aqueducthq.com/api-reference/sdk-reference/package-aqueduct/package-aqueduct.constants/aqueduct.constants.metrics)
-###\ package aqueduct.integrations
+### package aqueduct.integrations
```
Comment (vsreekanti) [inline api-reference/sdk-reference/package-aqueduct/README.md:18]: Same here.

**Review by vsreekanti — COMMENTED** [review]
@cw75, this should be good to go. Don't we have new artifact types as of this release that should be included as well?

**Comment by vsreekanti** [comment]
And haven't we deprecated check and metric artifacts?

**Comment by vsreekanti** [comment]
If there are other changes, please ping me before merging.

**Comment by cw75** [comment]
@vsreekanti we deprecated metric and check artifacts, but metric and check operators still exist, so I think the existing documentation is still relevant. And although we added other artifact types, they are currently all encapsulated into a type called `GenericArtifact` because we don't have type-specific syntax sugar yet. Once we do we can put them into new types.

**Pull request merged**